.gitmodules

```diff
@@ -0,0 +1,21 @@
+[submodule "layers/poky"]
+	path = layers/poky
+	url = https://git.yoctoproject.org/git/poky
+[submodule "layers/meta-openembedded"]
+	path = layers/meta-openembedded
+	url = https://github.com/openembedded/meta-openembedded.git
+[submodule "resin-yocto-scripts"]
+	path = resin-yocto-scripts
+	url = https://github.com/resin-os/resin-yocto-scripts.git
+[submodule "layers/oe-meta-go"]
```

Comment:
It would be nice to use pyro so we can get rid of this submodule.

Reply:
The meta-iot2000 only supports morty

Reply:
I don't think that it's true: https://github.com/siemens/meta-iot2000/tree/master/meta-iot2000-bsp . And it seems to be confirmed that it should work with pyro too in https://github.com/siemens/meta-iot2000/issues/60 .